src/en/about/about.md

```diff
@@ -2,13 +2,13 @@
 
 ## About The Project
 
-Front Page|Post
----|---
-![main screen](/docs/en/about/main_screen.png)|![chat screen](/docs/en/about/chat_screen.png)
+| Front Page                                     | Post                                           |
+| ---------------------------------------------- | ---------------------------------------------- |
+| ![main screen](/docs/en/about/main_screen.png) | ![chat screen](/docs/en/about/chat_screen.png) |
 
 [Lemmy](https://github.com/LemmyNet/lemmy) is similar to sites like [Reddit](https://reddit.com), [Lobste.rs](https://lobste.rs), [Raddle](https://raddle.me), or [Hacker News](https://news.ycombinator.com/): you subscribe to forums you're interested in, post links and discussions, then vote, and comment on them. Behind the scenes, it is very different; anyone can easily run a server, and all these servers are federated (think email), and connected to the same universe, called the [Fediverse](https://en.wikipedia.org/wiki/Fediverse).
 
-*The word “fediverse” (federated universe) refers to the network of all Lemmy servers [and other projects](https://blog.joinmastodon.org/2018/06/why-activitypub-is-the-future/), users of which are able to talk to each other seamlessly.*
+_The word “fediverse” (federated universe) refers to the network of all Lemmy servers [and other projects](https://blog.joinmastodon.org/2018/06/why-activitypub-is-the-future/), users of which are able to talk to each other seamlessly._
```

Comment:
Any reason to change asterisk to underscore? Personally I prefer the former.

Reply:
I also do honestly, but Prettier ( the markdown formatter used here ), is probably following the commonmark spec where `_` is the preferred option.